pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -0,0 +1,21 @@
+package bootstrap
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+// verifyBootstrapInPlace validate the number of control plane replica is one and that installation disk is set
+func verifyBootstrapInPlace(installConfig *types.InstallConfig) error {
```

Comment:
This comment got lost when the last update was made:

In other places in the validation code the validation framework is used to indicate exactly what field name is missing or has a bad value. Can that framework be used here, or do we need to update the error messages to be more specific? (See pkg/types/baremetal/validation/platform.go for an example)

Reply:
I'm not sure that I understand,
I don't see a difference between these error messages and the errors in  pkg/types/baremetal/validation/platform.go.
Are you suggesting to move pkg/asset/ignition/bootstrap/bootstrap_in_place.go to pkg/types/none/validation/ ?

Reply:
Eran and I talked about this and I'm fine with using another PR to touch up the error messages to be clearer about exactly which field has the bad value.

Reply:
@eranco74 I don't have permission to resolve threads, so feel free to close this one when you have a minute.